eng/Subsets.props

```
@@ -380,6 +381,10 @@
       Test="true" Category="clr" Condition="'$(DotNetBuildSourceOnly)' != 'true' and '$(NativeAotSupported)' == 'true'"/>
   </ItemGroup>
 
+  <ItemGroup Condition="$(_subset.Contains('+tools.cdacreader+'))">
+    <ProjectToBuild Include="$(RepoRoot)\src\native\managed\compile-native.proj" Category="tools" />
```

nit: match line 389's style
```suggestion
    <ProjectToBuild Include="$(SharedNativeRoot)managed\compile-native.proj" Category="tools" />
```